### ISSUE TITLE
[add to app][multiple_flutters]how to pass params to Flutter module

### ISSUE THREAD

**Issue by xuyisheng** [flutter/flutter#99257]
how can I pass params from Native to Flutter with Flutter Engine Groups API?

@pragma('vm:entry-point')
void topMain() => runApp(const MyApp(color: Colors.green));

for example. I want to run with Flutter "topMian" with a param "BookId"

➜  ~ flutter doctor
[✓] Flutter (Channel stable, 2.8.1, on Mac OS X 10.15.7 19H2 darwin-x64, locale
    zh-Hans-CN)
[✓] Android toolchain - develop for Android devices (Android SDK version
    31.0.0-rc4)

**Comment by darshankawar** [comment]
@xuyisheng 
I suggest you to ask this question on support channels like StackOverflow since this is more of a how to question.
If you don't get any solution and would like this to be added into add2app documentation, write in comments and I'll reopen it.

**Comment by xuyisheng** [comment]
I searched in StackOverflow and I did not find any solution, there is a 'options' param in @pragma('vm:entry-point'), but there is no way to pass param in Java/Kotlin code, pls show me how to do it, and added to the documentation

**Comment by darshankawar** [comment]
Thanks for the feedback. Reopening.

**Comment by darshankawar** [comment]
Can you take a look at [this](https://github.com/flutter/samples/tree/master/add_to_app/multiple_flutters) github examples and see if they help ?

**Comment by xuyisheng** [comment]
> Can you take a look at [this](https://github.com/flutter/samples/tree/master/add_to_app/multiple_flutters) github examples and see if they help ?

yes, this is the example I'm referring to, but it doesn't pass any parameters when launching the FlutterActivity

**Comment by xuyisheng** [comment]
> > Can you take a look at [this](https://github.com/flutter/samples/tree/master/add_to_app/multiple_flutters) github examples and see if they help ?
> 
> yes, this is the example I'm referring to, but it doesn't pass any parameters when launching the FlutterActivity


code in the demo:

```
EngineBindings(activity = this, delegate = this, entrypoint = "main")
```

there is no way to pass any parameters

**Comment by xuyisheng** [comment]
Any progress?

**Comment by xuyisheng** [comment]
> 


Any plan to support it ,i was waiting this day after day
